app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split()
```

Comment:
The variable name `command_ls` is not descriptive. Consider renaming it to something more meaningful, such as `command_parts` or `command_args`.